pinecone/client_test.go

```diff
@@ -66,8 +68,11 @@ func (ts *ClientTests) TestNewClientParamsSet() {
 	if client.sourceTag != "" {
 		ts.FailNow(fmt.Sprintf("Expected client to have empty sourceTag, but got '%s'", client.sourceTag))
 	}
+	if client.headers != nil {
+		ts.FailNow(fmt.Sprintf("Expected client to have nil headers, but got '%v'", client.headers))
```

Comment:
Total nit, but when I read this, I ask myself "what are nil headers" because it sounds like a quantity. Maybe rephrase this stmt to be `"Expected client headers to be nil"`.